### PR TITLE
Create General Info Page

### LINKED ISSUE #13: [HTML-CSS] Create general-info  page
Adding the required html and css for the general info page

**Note: the html is a sample for JS work, so please leave descriptive comments**

### PR DESCRIPTION
**Pull request by basillubbad**
Relates #13
This is a sample for the JS work

### CODE REVIEW

File: generalInfoPage/general.html [hunk 1]
```diff
@@ -21,7 +21,32 @@
         </nav>
     </header>
 
+    <section class="info-section">
+      <img class="country-img" src="/img/Turkey.png" alt="">
```
Comment (yaqootturman) [inline generalInfoPage/general.html]: name of the image file, not camelCase as it should

File: generalInfoPage/general.html [hunk 2]
```diff
@@ -21,7 +21,32 @@
...
+          <div > <p class="country-info"><b>Turkey</b></p></div> 
+          <div class="row-info"> <i class="fas fa-money-bill-wave icons"></i><p class="P-icon"> Turkish-lira</p></div> 
+          <div class="row-info">  <i class="far fa-comment-dots icons"></i><p class="P-icon">Turkish</p></div>
+         <div class="row-info">  <i class="fas fa-phone-alt icons"></i><p class="P-icon">+90</p></div> 
+           <div class="row-info"><i class="fas fa-globe-americas icons"></i><p class="P-icon">Asia</p></div> 
+        </div>
+        <button class="btn-universities"><a class="btn-a" href="/Universities Page/Universities.html">Go to universities
+                list</a></button>
+</div>
+    </section>
```
Comment (yaqootturman) [inline generalInfoPage/general.html:33]: a lot of white spacing, delete them,please

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: generalInfoPage/general.html (hunk 2 above)
Comment (basillubbad) [reply generalInfoPage/general.html:33]: done

File: generalInfoPage/general.html (hunk 1 above)
Comment (basillubbad) [reply generalInfoPage/general.html]: done

**Review by nada-abuzaid — APPROVED** [review]

File: generalInfoPage/general.html (hunk 1 above)
Comment (yaqootturman) [reply generalInfoPage/general.html]: You change the file name but you didn't change it in the src line

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: generalInfoPage/general.html (hunk 1 above)
Comment (basillubbad) [reply generalInfoPage/general.html]: done

**Review by yaqootturman — APPROVED** [review]

**Review by Yousef-Dall — APPROVED** [review]

**Pull request merged**